sources/slack/__init__.py

```diff
@@ -0,0 +1,76 @@
+"""Fetches Slack Orders and Products."""
+
+from typing import Iterable, Optional
+
+import dlt
+from dlt.common.time import ensure_pendulum_datetime
+from dlt.common.typing import TAnyDateTime, TDataItem
+from dlt.extract.source import DltResource
+
+from .helpers import SlackApi
+from .settings import MAX_PAGE_SIZE
+
+
+@dlt.source(name="slack")
+def slack_source(
+    access_token: str = dlt.secrets.value,
+    page_size: int = MAX_PAGE_SIZE,
+    created_before: Optional[TAnyDateTime] = None,
```

Comment:
should we support same interface we have in zendesk and shopify? so you can specify both start and end range. and use incremental for incremental loading, not replace
btw. we have a new quick with fully working incremental snippet
https://dlthub.com/docs/getting-started